src/components/ActionLink.tsx

```diff
@@ -0,0 +1,32 @@
+import type { ReactEventHandler } from "react"
+import { createUseStyles } from "react-jss"
+
+interface Props {
+  children: React.ReactNode
+  className?: string
+  onClick?: ReactEventHandler
+}
+
+const useStyles = createUseStyles({
+  button: {
+    background: "none",
+    border: "none",
+    color: "#1d70b8",
+    cursor: "pointer",
+    padding: "0",
+    textAlign: "left",
+    textDecoration: "underline"
+  }
+})
+
+const ActionLink: React.FC<Props> = ({ children, className, onClick }) => {
```

Comment:
You could do `const ActionLink = ({ children, className, onClick }: Props) => {` here, it might be easier to read? Tbh it's a preference thing.